src/main/java/org/jitsi/videobridge/Endpoint.java

```diff
@@ -489,16 +494,30 @@ public void send(PacketInfo packetInfo, String sourceEpId)
         else if (packet instanceof RtcpSrPacket)
         {
             RtcpSrPacket rtcpSrPacket = (RtcpSrPacket) packet;
-            if (bitrateController.transformRtcp(rtcpSrPacket))
+            AbstractEndpoint sourceEndpoint = getConference().getEndpoint(source);
```

Comment:
so we have to look up the endpoint and the media type of the ssrc foreach packet foreach sender?

Reply:
Well, it's limited to SRs, but you're right that's not great. I didn't like any of the alternatives, but something you mentioned in the jmt PR gave an idea. If nothing else I'll move this logic a step back to Conference so that it executes only once per packet.